game/settings/settings.h

```diff
@@ -24,11 +36,14 @@ struct DebugSettings {
   std::vector<DebugTextFilter> text_filters = {};
   bool text_check_range = false;
   float text_max_range = 0;
+
   u32 hide_imgui_key = SDLK_LALT;
 
-  void load_settings();
+  std::vector<KeyWithModifiers> toggle_fullscreen_keys = {
+      KeyWithModifiers(SDLK_RETURN, InputModifiers(SDL_KMOD_ALT)) void load_settings();
```

Comment:
I'm not sure if having multiple binds for this one functionality is necessary, if it's already configurable, just having one is fine.

Also, looks like you are not properly closing the bracket here.

Reply:
Yeah I made a mistake when I removed the other default of F12, Ive since resolved this error. Is there a benefit to moving this to a singular key as opposed to *potentially* multiple binds if the user wants?

For me it seems to make sense to support potential multiple binds, but default to 1. This way if the user wants more for whatever reason we can always dynamically support however many. Im not aware of a big tradeoff pro/con with switching to singular bind support. (I would want to add both locally myself)

Reply:
In general, I'd say having more options is not a bad thing, but in this case, I think it might cause more confusion/complications.

With your current changes, this is the only bind that works this way, which is inconsistent with the rest of the options and would just make it more confusing for anyone who wants to edit their binds and wonders why only the fullscreen bind supports multiple possible bindings. While we could probably make it so all binds support multiple variants, that would not only be extra work, it might also potentially break users' already existing custom binds.